Packs/Thycotic/Integrations/Thycotic/Thycotic.yml

```
@@ -0,0 +1,1096 @@
+category: Authentication
+commonfields:
+  id: Thycotic
+  version: -1
+configuration:
+- display: Server URL (e.g. https://example.net)
+  name: url
+  required: true
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  hidden: false
+  name: proxy
+  required: false
+  type: 8
+description: Secret Server is the only fully featured Privileged Account Management
+  (PAM) solution available both on premise and in the cloud. It empowers security
+  and IT ops teams to secure and manage all types of privileged accounts and offers
+  the fastest time to value of any PAM solution.
+display: Thycotic
+name: Thycotic
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: Secret ID
+      isArray: false
+      name: secret_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieved password from secret
+    execution: false
+    name: thycotic-secret-password-get
+    outputs:
+    - contextPath: Thycotic.Secret.Password
+      description: Password from secret
+      type: String
+  - arguments:
+    - default: false
+      description: Secret ID
+      isArray: false
+      name: secret_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieved username from secret
+    execution: false
+    name: thycotic-secret-username-get
+    outputs:
+    - contextPath: Thycotic.Secret.Username
+      description: Username from secret
+      type: String
+  - arguments:
+    - default: false
+      description: Search for secret using secret name.
+      isArray: false
+      name: search_name
+      required: true
+      secret: false
+    deprecated: false
+    description: Search ID secret by Name
```

Let's rephrase this one as well. How about this one?
```suggestion
    description: Search a secret by its name.
```

---

Done